input/data/manure/refreshed_manure_management.json

```
@@ -1,15 +1,39 @@
 {
-    "anaerobic_digesters":[
+    "anaerobic_digester":[
         {
-            "name": "anaerobic_digester",
+            "name": "anaerobic_digester1",
+            "digester_type": "anaerobic digester",
             "hydraulic_retention_time": 25,
             "top_cover_volume_fraction": 0.2,
             "evaporation_fraction": 0.02,
             "anaerobic_digestion_temperature_set_point": 37.5,
-            "methane_leakage_fraction": 0.01,
-            "next_steps": {
-                "storage": 1.0
-            }
+            "methane_leakage_fraction": 0.01
         }
+    ],
+    "separator":[
+        {
+            "name": "separator1",
+            "separator_type": "rotary screen",
+            "separated_solids_dry_matter": 0.20,
+            "total_solids_efficiency": 0.35,
+            "volatile_solids_efficiency": 0.40,
+            "nitrogen_efficiency": 0.30,
+            "ammoniacal_nitrogen_efficiency": 0.15,
+            "phosphorus_efficiency": 0.40,
+            "potassium_efficiency": 0.15,
+            "ash_efficiency": 0.10
```

```suggestion
            "ash_efficiency": 0.20
```
Thank you for confirming this! I should have provided this more explicitly. The default value (0.20) is the same for both types of separators.